O/osslsigncode/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "osslsigncode"
+version = v"2.10.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mtrojnar/osslsigncode",
+                  "4568c890cc1538ca80be3ee36775ba42223dea04"),
+]
+
+script = raw"""
+    cd $WORKSPACE/srcdir/osslsigncode
+
+    sed -i 's/Ws2_32/ws2_32/g' CMakeLists.txt
```

Comment:
Please submit a PR to fix this upstream, it's an upstream bug

Reply:
I opened a PR: https://github.com/mtrojnar/osslsigncode/pull/426